utility.h

```diff
@@ -184,6 +184,7 @@ class Utility
             return (double)timestamp / 1000.0;
             break;
         case TS_MICROS:
+        default:
```

Comment:
is this really intentional? It should move to be the last entry if that's the case.